### PR TITLE
[IMP] product_price_history : add new graph view

### PR DESCRIPTION
**Pull request by legalsylvain**
- Add the possibility to see a graph with the evolution of the cost of the product over the time.

product_price_history_graph

- add me as comaintainers

- add french translation

### CODE REVIEW

File: product_price_history/models/product_template.py [hunk 1]
```diff
@@ -4,6 +4,8 @@
 class ProductTemplate(models.Model):
     _inherit = 'product.template'
 
+    cost = fields.Float(group_operator="avg")
```
Comment (ghost) [inline product_price_history/models/product_template.py]: Is it a good idea to have the value of cost propagate to the product variant if there is only one variant ?

**Review by rousseldenis — APPROVED** [review]

File: product_price_history/models/product_template.py (hunk 1 above)
Comment (legalsylvain) [reply product_price_history/models/product_template.py]: Indeed. I have to take a look.

Comment (legalsylvain) [reply product_price_history/models/product_template.py]: Hi @gbrito-erpopen. I updated the code. I made an error, I wanted to set the group_operator on the ``cost`` field of the ``product.price.history`` model.
Now fixed.

**Review by ghost — APPROVED** [review]

File: product_price_history/models/product_template.py (hunk 1 above)
Comment (ghost) [reply product_price_history/models/product_template.py]: @legalsylvain looks good, makes more sense :)

**Comment by legalsylvain** [comment]
@rousseldenis could you call ocabot to merge this one ? 

thanks !

**Comment by legalsylvain** [comment]
thanks !

**Pull request merged**